docs/mobile-apps/virtual-usb.md

```diff
@@ -15,164 +15,247 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+* Use a mix of live and automated testing as it fits your use case.
+* Make the most out of your early stage development.
+* Build and deploy an app directly from your IDE.
+* Monitor device performance metrics such as CPU consumption, device memory, and network data performance.
 
-With vUSB, you can use homegrown developing and testing tools because it integrates into the development environment as if the device was directly connected to the workstation. When you debug your applications, vUSB provides the ability to build and deploy an application directly from the IDE. You can also directly monitor device performance metrics such as CPU consumption, device memory, and network data performance.
+This topic provides vUSB instructions for Sauce Labs and for TestObject, our [legacy RDC platform](https://wiki.saucelabs.com/display/DOCS/Legacy+Real+Device+Platform+Resources). To take advantage of our latest features, we recommend using Sauce Labs. For information on upcoming and available real device cloud features, refer to our [feature preview doc](https://wiki.saucelabs.com/pages/viewpage.action?pageId=102721844).
 
-This topic provides instructions for using vUSB with Sauce Labs and TestObject, our legacy real device cloud platform. To take advantage of our latest features, we recommend using Sauce Labs. For more information, see [Sauce Labs Real Device Testing Features](https://wiki.saucelabs.com/pages/viewpage.action?pageId=102721844) and [Legacy Real Device Platform Resources](https://wiki.saucelabs.com/display/DOCS/Legacy+Real+Device+Platform+Resources).
+## What You'll Need
 
->**NOTE**:Your access may depend on whether you're using the Legacy platform or the Sauce Labs platform. For more information on upcoming and available features for Real Devices, refer to the [feature preview doc](https://wiki.saucelabs.com/pages/viewpage.action?pageId=102721844).
+<p><Highlight color="#013a70">ENTERPRISE PLANS ONLY</Highlight></p>
 
-<img src={useBaseUrl('img/virtual-usb/vusb-workflow.png')} alt="Virtual USB Workflow" width="400" />
+* For security reasons, you'll need to have our [private real device cloud](https://saucelabs.com/platform/real-device-cloud) feature enabled as part of your [enterprise pricing plan](https://saucelabs.com/solutions/enterprise). This is a dedicated pool of Sauce Labs real devices allocated to your organization only.
+* Windows, Mac or Linux operating system.
+* Administrator and or installation rights on your machine.
+* Have Java Development Kit (JDK) installed.
+* If you want to use [Sauce Connect Proxy](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy), have the client installed.
+* If you're testing Android apps:
+  * Android Debug Bridge (ADB) version higher than 1.0.39.
+  * Android Studio 4 or higher.
 
+## Virtual USB for Sauce Labs
 
-## What You'll Need
+<img src={useBaseUrl('img/virtual-usb/vusb-workflow.png')} alt="Virtual USB Workflow" width="800" />
```

Comment:
There are 2 workflows:
- starting a new session from the command line
- connecting to an existing session

This image only covers one and might be unclear for people. Secondly I miss a story for this image. It's just an image, it's not telling me a lot.

My advice would be to have or both workflows in images and have a story with it, or remove it

Reply:
That makes sense -- I'll remove it for now. It would be cool to do a second workflow, once I have bandwidth.